config/environments/production.rb

```diff
@@ -38,8 +37,25 @@
   # config.action_cable.url = "wss://example.com/cable"
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  config.assume_ssl = true
+
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
+
+  # Log to STDOUT by default
+  config.logger = ActiveSupport::Logger.new($stdout)
+    .tap  { |logger| logger.formatter = ::Logger::Formatter.new }
+    .then { |logger| ActiveSupport::TaggedLogging.new(logger) }
+
+  # Prepend all log lines with the following tags.
+  config.log_tags = [:request_id]
```

Comment:
This is still draft but just in case its helpful to point out, there is some default logging configuration in config/application.rb and from what I remember Rails' `log_tags` feature does not work with lograge and structured/json logging because it mangles the formatting by prepending the tags outside of the provided formatter like `[rails log tags] {then the log message}`. We've configured lograge to include the request_id and other useful things so log_tags shouldn't be necessary (unless we overhaul things a bit): https://github.com/alphagov/forms-api/blob/main/app/controllers/application_controller.rb#L23

Reply:
I wasn't aware of that. I'll remove this incoming change from the Rails upgrade.